### PR TITLE
263625: Added drill down for location on the block level

### PR DESCRIPTION
**Pull request by pstudzinski**
Added drill down for location on the block level

### CODE REVIEW

File: custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 /* global d3 */
 
-function MapOrSectorController() {
+function MapOrSectorController($scope, $location, storageService, locationsService) {
```
Comment (lwyszomi) [inline custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js]: ```$scope``` is not used in the code. Please remove it.

File: custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js [hunk 2]
```diff
@@ -82,11 +97,11 @@ function MapOrSectorController() {
     };
 }
 
-MapOrSectorController.$inject = [];
+MapOrSectorController.$inject = ['$scope', '$location', 'storageService', 'locationsService'];
```
Comment (lwyszomi) [inline custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js]: $scope is not used in the code. Please remove it.

File: custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js (hunk 1 above)
Comment (pstudzinski) [reply custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js]: removed

File: custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js (hunk 2 above)
Comment (pstudzinski) [reply custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js]: removed

**Pull request merged**